src/code/z_actor.c

```diff
@@ -1532,41 +1514,44 @@ s32 func_8002F1C4(Actor* actor, GlobalContext* globalCtx, f32 arg2, f32 arg3, u3
     return true;
 }
 
-s32 func_8002F298(Actor* actor, GlobalContext* globalCtx, f32 arg2, u32 exchangeItemId) {
-    return func_8002F1C4(actor, globalCtx, arg2, arg2, exchangeItemId);
+s32 Actor_RequestToTalkAndExchangeItem(Actor* actor, GlobalContext* globalCtx, f32 range, u32 exchangeItemId) {
+    return Actor_RequestToTalkAndExchangeItemInRange(actor, globalCtx, range, range, exchangeItemId);
 }
 
-s32 func_8002F2CC(Actor* actor, GlobalContext* globalCtx, f32 arg2) {
-    return func_8002F298(actor, globalCtx, arg2, EXCH_ITEM_NONE);
+s32 Actor_RequestToTalkInRange(Actor* actor, GlobalContext* globalCtx, f32 radius) {
+    return Actor_RequestToTalkAndExchangeItem(actor, globalCtx, radius, EXCH_ITEM_NONE);
 }
 
-s32 func_8002F2F4(Actor* actor, GlobalContext* globalCtx) {
-    f32 var1 = 50.0f + actor->colChkInfo.cylRadius;
+s32 Actor_RequestToTalk(Actor* actor, GlobalContext* globalCtx) {
+    f32 radius = 50.0f + actor->colChkInfo.cylRadius;
 
-    return func_8002F2CC(actor, globalCtx, var1);
+    return Actor_RequestToTalkInRange(actor, globalCtx, radius);
 }
 
-u32 func_8002F334(Actor* actor, GlobalContext* globalCtx) {
+u32 Actor_IsDoneTalking(Actor* actor, GlobalContext* globalCtx) {
```

Comment:
Wording is weird here. Maybe `Actor_HasFinishedTalking`?
Also, the same question about `u32` vs `s32` here.